lib/handler/status/durations.c

```diff
@@ -64,6 +69,12 @@ static void durations_status_per_thread(void *priv, h2o_context_t *ctx)
         ADD_DURATION(response_time);
         ADD_DURATION(total_time);
 #undef ADD_DURATION
+
+#ifndef H2O_USE_LIBUV
```

Comment:
My apologies, this is incorrect. `H2O_USE_LIBUV` is always defined. Check here should be `#if !H2O_USE_LIBUV`.

Reply:
Nice catch!

I've clarified our convention in https://github.com/h2o/h2o/wiki/Coding-Style#conditional-compilation.